general_superstaq/user_config_test.py

```diff
@@ -80,6 +80,20 @@ def test_ibmq_set_token(  # pylint: disable=missing-function-docstring
     assert service.ibmq_set_token("valid token") == "Your IBMQ account token has been updated"
 
 
+@mock.patch(
+    "general_superstaq.superstaq_client._SuperstaQClient.cq_set_token",
+    return_value="Your CQ account token has been updated",
+)
+def test_mq_set_token(  # pylint: disable=missing-function-docstring
+    mock_cq: mock.MagicMock,
```

Comment:
(assuming prior suggestion)
```suggestion
    mock_post_request: mock.MagicMock,
```